daprdocs/content/en/reference/components-reference/supported-bindings/wasm.md

```diff
@@ -80,9 +83,23 @@ pass metadata properties with each request:
 
 - `args` any CLI arguments, comma-separated. This excludes the program name.
 
-For example, if the binding `url` was a Ruby interpreter, such as from
-[webassembly-language-runtimes](https://github.com/vmware-labs/webassembly-language-runtimes/releases/tag/ruby%2F3.2.0%2B20230215-1349da9),
-the following request would respond back with "Hello, salaboy":
+For example, consider binding the `url` to a Ruby interpreter, such as from
+[webassembly-language-runtimes](https://github.com/vmware-labs/webassembly-language-runtimes/releases/tag/ruby%2F3.2.0%2B20230215-1349da9):
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: wasm
+spec:
+  type: bindings.wasm
+  version: v1
+  metadata:
+  - name: url
+    value: "https://github.com/vmware-labs/webassembly-language-runtimes/releases/download/ruby%2F3.2.0%2B20230215-1349da9/ruby-3.2.0-slim.wasm"
+```
```

Comment:
I was not able to test this snippet though (see discord thread)

Reply:
I have updated the documentation to actually reflect the implementation. The issue is the original example was tokenizing naively at `,`, causing the following arglist to be split incorrectly:

```
    "args": "-ne,'print \"Hello, \"; print'" => `-ne` `'print "Hello` ` "; print'`
```

a working example is instead:

```
    "args": "-ne,print \"Hello \"; print" => `-ne` `print "Hello "; print`
```

notice how quotes are currently being ignored. This is obviously suboptimal and should be fixed :)